app/tests/spec/views/change_password.js

```diff
@@ -222,7 +222,14 @@ function (chai, _, $, sinon, AuthErrors, FxaClient, Metrics, p,
               assert.isTrue(view.fxaClient.changePassword.calledWith(
                   EMAIL, oldPassword, newPassword));
               assert.isTrue(view.fxaClient.signIn.calledWith(
-                      EMAIL, newPassword, relier));
+                  EMAIL,
+                  newPassword,
+                  relier,
+                  {
+                    reason: view.fxaClient.SIGNIN_REASON.PASSWORD_CHANGE,
+                    sessionTokenContext: account.get('sessionTokenContext')
```

Comment:
If you look at the original test, we were not checking the options block. When two objects are compared, all attributes must be compared. A test will fail if the expected object contains a single attribute and the actual object contains 2. For the test to pass, all attributes must be checked.